indexing-service/pom.xml

```diff
@@ -90,6 +90,16 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
+      <dependency>
+        <groupId>org.apache.hadoop</groupId>
+        <artifactId>hadoop-common</artifactId>
```

Comment:
We really shouldn't add hadoop dependencies to druid core because 1) it's unnecessary if we don't use hadoop and 2) it might cause some errors because of the version mismatch.

Please move them to `indexing-hadoop`. The version must not be specified and the scope should be `provided`.